ui/store/actions.ts

```diff
@@ -704,7 +704,7 @@ export function decryptMsg(
     }
 
     dispatch(updateMetamaskState(newState));
-    dispatch(completedTx(decryptedMsgData.metamaskId));
+    dispatch(completedTx(decryptedMsgData.metamaskId as unknown as string));
```

Comment:
If we're casting here, does that imply we have to update a type inside `completedTx`?

Reply:
Good call, after looking at it second time, actually we don't need this casting since messageId/metamaskId is actually a string on `XMessages`.
Fixed in  https://github.com/MetaMask/metamask-extension/pull/20952/commits/837f1308c62c1a52e64da2e9f1ded6e1083f4c43